tests/examples/shims/awesome.lua

```diff
@@ -48,16 +46,19 @@ awesome.startup = true
 function awesome.register_xproperty()
 end
 
-local init, surfaces = false, {}
+-- local init, surfaces = false, {}
 
 function awesome.load_image(file)
+    -- The code below is actually the closest to what Awesome really does, but
+    -- it is creating a GDK warning when running without a display server (but
+    -- still works)
```

Comment:
Oh and: How did this end up in git? Shouldn't the commit that added this code already have failed?

Reply:
An attempt at breaking up a huge PR you (rightfully) rejected as too big ended up creating a semi tangled mess of smaller PRs. It was merged as part of a squashed set of other shims commit but wasn't used at that time. As it works perfectly when an X11 server is present, it went unnoticed.